src/main/java/racing/CarRace.java

```diff
@@ -0,0 +1,37 @@
+package racing;
+
+import java.util.List;
+import java.util.Random;
+
+public class CarRace {
+    public static final int RANDOM_BOUND = 10;
+    public static final int MOVE_THRESHOLD = 4;
+    private Random randomGenerator;
```

Comment:
상수와 인스턴스 변수의 구분을 위해서 공백 라인을 추가해보면 어떨까요?
```suggestion
    public static final int MOVE_THRESHOLD = 4;
    
    private Random randomGenerator;
```